### DISCUSSION TITLE
Drag and drop events in calendar into external drop zone

### DISCUSSION THREAD

**Discussion by zokir14** [antoniandre/vue-cal#3] [Q&A] [answered]
is it possible to drag and drop an event in calendar into draggable drop zone?

**Comment by antoniandre** [comment, marked as answer]
Yes the calendar event is a pure HTML draggable: you can use events like @dragenter, @dragover, @dragleave, @drop to implement it and receive the event information.
To retrieve on drop you should be able to use `e.dataTransfer.getData('event')`

Cheers
